tools/schemacode/bidsschematools/validator.py

```diff
@@ -845,10 +845,13 @@ def validate_bids(
 
     Examples
     --------
-    >>> from bidsschematools import validator
-    >>> bids_paths = '~/.data2/datalad/000026/rawdata'
-    >>> schema_version='{module_path}/data/schema/'
-    >>> validator.validate_bids(bids_paths, schema_version=schema_version)"
+
+    ::
```

Comment:
I'm not sure if I prefer this or the more verbose `>>>`, but still very cool.

Reply:
The `>>>` means the code gets executed in doctests, but this test fails on anybody but @TheChymera's computer.

Reply:
Ah, well we aren't all as blessed to own a computer like @TheChymera thanks for adding this.